exasol_script_languages_container_tool/lib/tasks/upload/upload_container_base_task.py

```diff
@@ -85,34 +84,30 @@ def generate_command_line_output_str(
         )
         return command_line_output_str
 
-    def _upload_container(self, release_info: ExportInfo):
-        s = requests.session()
-        url = self._get_upload_url(release_info, without_login=True)
-        self.logger.info(f"Upload {release_info.cache_file} to {url}")
-        with open(release_info.cache_file, "rb") as file:
-            response = s.put(url, data=file, auth=self._create_auth_object())
-            response.raise_for_status()
-
-    def _create_auth_object(self) -> HTTPBasicAuth:
-        auth = HTTPBasicAuth(self.bucketfs_username, self.bucketfs_password)
-        return auth
+    def build_file_path_in_bucket(self, release_info: ExportInfo) -> bfs.path.PathLike:
+        backend = bfs.path.StorageBackend.onprem
 
-    def _get_upload_url(self, release_info: ExportInfo, without_login: bool = False):
         complete_release_name = self._get_complete_release_name(release_info)
-        if without_login:
-            login = ""
-        else:
-            login = f"""{self.bucketfs_username}:{self.bucketfs_password}@"""
-        path_in_bucket = (
-            f"{self.path_in_bucket}/" if self.path_in_bucket not in [None, ""] else ""
-        )
-        url = (
-            f"""{self._get_url_prefix()}{login}"""
-            + f"""{self.database_host}:{self.bucketfs_port}/{self.bucket_name}/{path_in_bucket}"""
-            + complete_release_name
-            + ".tar.gz"
-        )
-        return url
+        kwargs = {
```

Comment:
I think, long-term we need to extract the following out of PEC into bucketfs python or so to simplify the backend inference 

https://github.com/exasol/python-extension-common/blob/822762ae0359744a21a6f485bad675fbe281f9d7/exasol/python_extension_common/deployment/language_container_deployer.py#L312

Reply:
Created https://github.com/exasol/bucketfs-python/issues/160